addons/l10n_se/i18n/l10n_se.pot

```diff
@@ -192,3 +192,8 @@ msgid ""
 "You can choose different models for each type of reference. The default one "
 "is the Odoo reference."
 msgstr ""
+
+#. module: l10n_se
+#: model:ir.ui.menu,name:l10n_se.account_reports_se_statements_menu
+msgid "Sweden K3"
```

Comment:
This is not sorted alphabetically

You can have a look at https://man7.org/linux/man-pages/man1/msgcat.1.html